recipes/apache.rb

```diff
@@ -15,7 +15,7 @@
             password: node['kibana']['apache']['basic_auth_password'])
   owner node['apache']['user']
   group node['apache']['group']
-  mode 00644
+  mode 0o0644
```

Comment:
Offenses:

recipes/apache.rb:18:8: C: Use 0o for octal literals.
  mode 00644
       ^^^^^